src/scss/_header.scss

```diff
@@ -241,7 +241,7 @@
     }
 
     .mobile-menu {
-        @apply hidden w-full;
+        @apply hidden w-full h-screen overflow-y-auto pb-16;
```

Comment:
Ah! Now I see. It's accounting for the height of the logo portion _above_ the mobile menu, which, because the menu has a height of 100% of the _viewport_, pushes the lower portion of the menu out view (thus hiding the Get Started button). This additional bit of padding creates just enough space below the button to allow you to scroll to it without having to get to the bottom of the page first.

Definitely works, so 👍 on the implementation for now. But since it took me a bit to figure out where that additional 4 rems of padding was "disappearing" to, a comment explaining what's going on would probably help (and keep someone like me from making a change that messes things up around it).

Reply:
👍 Add a comment to clarify why it's there.